ee/api/test/__snapshots__/test_organization_resource_access.ambr

```diff
@@ -48,7 +48,7 @@
          "posthog_organization"."available_features"
   FROM "posthog_organization"
   WHERE "posthog_organization"."id" = '00000000-0000-0000-0000-000000000000'::uuid
-  LIMIT 21 /*controller='organization_resource_access-list',route='api/organizations/%28%3FP%3Cparent_lookup_organization_id%3E%5B%5E/.%5D%2B%29/resource_access/%3F%24'*/
```

Comment:
I don't remember where they're set but it looks like the useful SQL commenting has stopped working

Reply:
reasonably sure it's the sql commenter: https://github.com/PostHog/posthog/blob/master/posthog/settings/web.py#L90

Reply:
oh dang it!

It's pretty useful for knowing where the request is coming from, specially when looking in, say, pganalyze for most popular sources of sql requests.

But, we do this manually for clickhouse ourselves, so wonder if there's an easy / small-scoped way to _just do it_ ourselves

Reply:
I've investigated further and also tried to use the middleware from opentelemetry that has some modifications as it seems.
Unfortunately, then immediately the following issue comes up 🥴 
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1554

It seems there are some differences between how the queries are handled and how the wrappers are executed. This would also explain why the `pg_sleep` wrapper has no effect anymore.

I'd say for now I'll make another PR just to upgrade to Django 4.1, where I had all CI green. Or does anyone have any suggestions?

Reply:
Yep... a step on the road that works 🚀